contract/.solhint.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I noticed that if `.solhint.json` is in `contracts` folder, my VSCode doesn't pick it up.

By moving this file into the root of the repository, it works for VSCode + `yarn lint` still works properly too.

Reply:
Thanks Brian! I've pushed your suggested fix for the prettier / linting issue 👍  Feel free to doublecheck the issue is resolved in your end now

Reply:
Thanks, looks good!